### PR TITLE
how to enable automated unhealthy nodes detector on azure

### PR DESCRIPTION
**Pull request by whites11**
towards: https://github.com/giantswarm/giantswarm/issues/14186

This PR adds instructions on how to enable and disable the automated unhealthy node feature on azure clusters.

### CODE REVIEW

File: src/content/basics/automatic-termination-of-bad-nodes/index.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ categories: ["basics"]
 
 Degraded nodes in a Kubernetes cluster should be a rare issue, however when it occurs, it can have severe consequences for the workloads scheduled to the affected nodes. The goal should be to detect bad nodes early and remove them from the cluster, replacing them with healthy ones.
 
-Starting with tenant cluster release v12.6.0 for AWS, you now have the option to automate the detection and termination of bad nodes. When enabled, all nodes in your cluster are periodically checked. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
+Starting with tenant cluster release v12.6.0 for AWS and v13.1.0 for Azure, you now have the option to automate the detection and termination of bad nodes. When enabled, all nodes in your cluster are periodically checked. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
 
-This function is available on AWS only currently. Support on other providers will follow.
+This function is currently available on AWS and Azure only. Support on KVM provider will follow.
```
Comment (marians) [inline src/content/basics/automatic-termination-of-bad-nodes/index.md:16]:
```suggestion
This function is currently available on AWS and Azure only.
```

File: src/content/basics/automatic-termination-of-bad-nodes/index.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ An unhealthy status means the `kubelet` on a given node has reported `NotReady`
 
 Automatic termination of unhealthy nodes is **disabled by default**. You can enable it for each individual cluster.
 
+This paragraph explains how you can enable the feature for each supported provider.
```
Comment (marians) [inline src/content/basics/automatic-termination-of-bad-nodes/index.md:28]:
```suggestion
This section explains how you can enable the feature for each supported provider.
```

File: src/content/basics/automatic-termination-of-bad-nodes/index.md [hunk 3]
```diff
@@ -46,3 +50,23 @@ spec:
...
+To enable it, you have to edit the [`Cluster`](/reference/cp-k8s-api/clusters.cluster.x-k8s.io/) resource of your cluster using the [Control Plane Kubernetes API](/basics/api/#cp-k8s-api).
+
+Make sure the resource has the `alpha.node.giantswarm.io/terminate-unhealthy` annotation. The value can be anything you like, as only the presence of that annotation is checked. Here is an example:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  annotations:
+    alpha.node.giantswarm.io/terminate-unhealthy: "true"
```
Comment (marians) [inline src/content/basics/automatic-termination-of-bad-nodes/index.md:65]: Is it required that the value here is `"true"`? If not, I would suggest to make it consistent with line 41. In which way you align is up to you.

Comment (whites11) [reply src/content/basics/automatic-termination-of-bad-nodes/index.md:65]: done

**Review by marians — APPROVED** [review]

**Pull request merged**